tests/OmniSharp.Roslyn.CSharp.Tests/BlockStructureFacts.cs

```diff
@@ -0,0 +1,58 @@
+using System.Linq;
+using System.Threading.Tasks;
+using OmniSharp.Models.v2;
+using OmniSharp.Roslyn.CSharp.Services.Structure;
+using OmniSharp.Roslyn.Extensions;
+using TestUtility;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OmniSharp.Roslyn.CSharp.Tests
+{
+    public class BlockStructureFacts : AbstractSingleRequestHandlerTestFixture<BlockStructureService>
+    {
+        public BlockStructureFacts(ITestOutputHelper output, SharedOmniSharpHostFixture sharedOmniSharpHostFixture)
+            : base(output, sharedOmniSharpHostFixture)
+        {
+        }
+
+        protected override string EndpointName => OmniSharpEndpoints.V2.BlockStructure;
+
+        [Fact]
+        public async Task UsesRoslynBlockStructureService()
+        {
+            var testFile = new TestFile("foo.cs", @"class Foo[|
+{
+    void M()[|
+    {
+        if (false)[|
+        {
+        }|]
+    }|]
+}|]");
+            var text = testFile.Content.Text;
+
+            var lineSpans = (await GetResponseAsync(testFile)).Spans
+                .Select(b => b.TextSpan)
+                .ToArray();
+
+
+            var expected = testFile.Content.GetSpans()
+                .Select(t => t.ToRange(text.Lines)).ToArray();
```

Comment:
Note that there's already a `TestContent.GetRangeFromSpan(...)` helper that you can use. Ultimately that calls an existing extension method in TestUtility on `SourceText`: https://github.com/OmniSharp/omnisharp-roslyn/blob/master/tests/TestUtility/SourceTextExtensions.cs.

Reply:
That extension returns `TestUtiltity.TextRange`, but my product code returns `OmniSharp.Models.V2.Range`. These types have essentially the same shape and there are even duplicated test/product `Point` classes. I don't really want to write code that knows how to compare test and product Range/Points. Thoughts on the best way to resolve this?